WorkloadGenerator/config.py

```diff
@@ -8,3 +8,7 @@
 AUDIT_SERVER_NAME = 'AuditService'
 AUDIT_SERVER_IP = 'localhost'
 AUDIT_SERVER_PORT = 6500
+
+TRIGGER_SERVER_NAME = 'TriggerService'
+TRIGGER_SERVER_IP = 'localhost'
+TRIGGER_SERVER_PORT = 7000

```

Comment:
Please confirm this port number is the same as in the shell config.sh script for the trigger service. If not, change the port number in the shell script to match this.